_pkgdown.yml

```diff
@@ -58,3 +58,4 @@ reference:
     desc: Functions deprecated in teal.data v0.3.1
     contents:
       - TealData
+      - get_labels
```

Comment:
We have much more functions in `R/deprecated.R` - should we add them all here?

Reply:
```suggestion
    contents:
      - TealData
      - get_labels
      - get_join_keys
```

I think only `get_join_keys` is missing

Reply:
All other the functions are documentated under `TealData` documentation. but my I forgot to mention `get_join_key()` here.

Reply:
I meant `teal_data_file`, `cdisc_data_file`, `get_cdisc_keys` and other functions defined here but you are right that all of them will be listed there via `TealData` entry 👍

Reply:
Shouldn't `get_join_keys` fall under `TealData` as well?

Reply:
We have separate documentation for `get_join_keys` as we are recommending you to use `join_keys()` instead.

Reply:
I know we do. That is the very reason that I ask. I can only presume it used to be used to get keys from `TealData`, because I cannot read it from the documentation currently on `main`.
![image](https://github.com/insightsengineering/teal.data/assets/114988527/9e3f1968-f11a-42e7-acc7-a95f71b15840)

Reply:
`get_cdisc_keys` and `set_keys` are in there.

Reply:
Now, I think the `get_join_keys` should be under `TealData`.  as user  need to migrate completely and understand `teal_data` as a whole. Perhaps we can include it in the documentation with minor changes in description, e.g. 

changes Description like This function has been deprecated. Please see [teal_data()](http://127.0.0.1:44533/help/library/teal.data/help/teal_data), [cdisc_data()](http://127.0.0.1:44533/help/library/teal.data/help/cdisc_data), and [join_keys()](http://127.0.0.1:44533/help/library/teal.data/help/join_keys) instead. Find more information on [Migration guide](https://github.com/insightsengineering/teal/discussions/945).
```

.deprecate_function <- function(what, details) {
  lifecycle::deprecate_stop(
    when = "0.3.1",
    what = what,
    details =details
  )
}

#' @rdname TealData
#' @export
get_join_keys <- function(...) {
.deprecate_teal_data("get_join_keys()", "Use `join_keys(data)` instead.")
}
```

WDYT? @chlebowa

Reply:
> This function has been deprecated. Please see [teal_data()](http://127.0.0.1:44533/help/library/teal.data/help/teal_data), [cdisc_data()](http://127.0.0.1:44533/help/library/teal.data/help/cdisc_data), and [join_keys()](http://127.0.0.1:44533/help/library/teal.data/help/join_keys) instead. Find more information on [Migration guide](https://github.com/insightsengineering/teal/discussions/945).

Actually, I am not too fond of this description. Perhaps a better one would be 
> The `TealData` class and associated functions have been deprecated. Use [`teal_data()`] instead.
See the [Migration Guide](https://github.com/insightsengineering/teal/discussions/945) for details.

How about it?

Reply:
I like this one better. It is more actionable.

Reply:
> How about it?

cdisc_data() is a wrapper around teal_data() but it would be more helpful to include this and join_keys()  information and if not in description then possibly add join_keys() under `@seealso`. ?

Reply:
Sure :+1:

Reply:
done [11a227a](https://github.com/insightsengineering/teal.data/pull/240/commits/11a227a1af1572569c27c33b777bed98f5b92b2c)